go/interpreter/lfvm/instructions_test.go

```diff
@@ -337,77 +320,36 @@ func TestBlobHash(t *testing.T) {
 				stack:  NewStack(),
 				memory: NewMemory(),
 			}
-			ctxt.gas = test.gas
-			ctxt.params.Revision = test.revision
+			ctxt.gas = 2
+			ctxt.params.Revision = tosca.R13_Cancun
 
 			test.setup(&ctxt.params, ctxt.stack)
 
 			err := opBlobHash(&ctxt)
-			if want, got := test.err, err; want != got {
-				t.Fatalf("unexpected return, wanted %v, got %v", want, got)
+			if err != nil {
+				t.Fatalf("unexpected return: %v", err)
 			}
-
-			if err == nil {
-				if want, got := test.want, ctxt.stack.data[0]; tosca.Hash(got.Bytes32()) != want {
-					t.Fatalf("unexpected value on top of stack, wanted %v, got %v", want, got)
-				}
+			if want, got := test.want, ctxt.stack.data[0]; tosca.Hash(got.Bytes32()) != want {
+				t.Fatalf("unexpected value on top of stack, wanted %v, got %v", want, got)
 			}
 		})
 	}
 }
 
-func TestBlobBaseFee(t *testing.T) {
-
-	blobBaseFeeValue := tosca.Value{1}
-
-	tests := map[string]struct {
-		setup    func(*tosca.Parameters)
-		gas      tosca.Gas
-		revision tosca.Revision
-		err      error
-		want     tosca.Value
-	}{
-		"regular": {
-			setup: func(params *tosca.Parameters) {
-				params.BlobBaseFee = blobBaseFeeValue
-			},
-			gas:      2,
-			revision: tosca.R13_Cancun,
-			want:     blobBaseFeeValue,
-		},
-		"old-revision": {
-			setup:    func(*tosca.Parameters) {},
-			gas:      2,
-			revision: tosca.R12_Shanghai,
-			err:      errInvalidRevision,
-			want:     tosca.Value{},
+func TestBlobBaseFee_ReturnsErrorWhenCalledWithUnsupportedRevision(t *testing.T) {
+	ctxt := context{
+		params: tosca.Parameters{
+			Recipient: tosca.Address{1},
 		},
+		stack:  NewStack(),
+		memory: NewMemory(),
 	}
+	ctxt.gas = 2
```

Comment:
Likely all those parameters except the Revision are not relevant for this test.

Reply:
Yes, I was able to remove several parameters.